libs/entitlement/qa/entitlement-test-plugin/src/main/java/org/elasticsearch/entitlement/qa/test/URLConnectionNetworkActions.java

```diff
@@ -218,4 +229,14 @@ static void baseUrlConnectionGetContentWithClasses() throws Exception {
     static void sunHttpConnectionGetContentWithClasses() throws Exception {
         withJdkHttpConnection(conn -> conn.getContent(new Class<?>[] { String.class }));
     }
+
+    @EntitlementTest(expectedAccess = PLUGINS)
+    static void sunMailToURLConnectionConnect() throws Exception {
+        withJdkMailToConnection(URLConnection::connect);
+    }
+
+    @EntitlementTest(expectedAccess = PLUGINS)
+    static void sunMailToURLConnectionGetOutputStream() throws Exception {
+        withJdkMailToConnection(URLConnection::connect);
```

Comment:
This should vall `getOutputStream` I think?

Reply:
Fixed!